instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -182,35 +232,78 @@ def get_or_create_headers():
                 exception = exc
                 result = getattr(exc, "response", None)
             finally:
-                elapsed_time = max(
-                    round((default_timer() - start_time) * 1000), 0
-                )
+                elapsed_time = max(default_timer() - start_time, 0)
```

Comment:
So, elapsed time is always seconds but then multiplied for the old histrogram. I like this approach